web/scripts/start-console.sh

```diff
@@ -80,6 +85,10 @@ function createEnvironment(){
     echo BRIDGE_PLUGINS=$BRIDGE_PLUGINS >> scripts/env.list
 }
 
+if [[ $CREATE_ENV == 1 ]] || [[ -f "scripts/env.list" ]]; then
```

Comment:
This will delete `scripts/env.list` regardless of the value of `$CREATE_ENV`. Is that what we want? I think I would be surprised if I ran the script without the create environment flag and it deleted the `env.list` file from the previous run.

Maybe the logic should be like below (not tested). Or am I missing the point?
```
if [[ $CREATE_ENV == 1 ]]; then
  if [[ -f "scripts/env.list" ]]; then
    removeEnvironment
  fi
  createEnvironment
fi
```

Reply:
That's a good point. My goal was to have the script check if I was logged into a different cluster and if so, update my env.list. I updated the PR to check if the previous cluster endpoint differs from the current cluster endpoint before removing the env.list.

Reply:
Thanks for the explanation. Now I see why it should also check when CREATE_ENV is 0.